Constants/EndPoint.js

```diff
@@ -1,7 +1,7 @@
 const { use } = require("../playwright.config");
 
 
-function EndPoints() {
+function  getApiEndPoints() {
     const activitiesEndPoint = use.baseURL+'/Activities';
```

Comment:
why you have created a function for this, you could have directly created some constants in a class and used them.